keras/utils/vis_utils.py

```diff
@@ -290,8 +290,7 @@ def format_shape(shape):
             [format_shape(ishape) for ishape in layer.input_shapes])
       else:
         inputlabels = '?'
-      label = '{%s}|{input:|output:}|{{%s}}|{{%s}}' % (label, inputlabels,
-                                                       outputlabels)
+      label = '{%s}|{input:|output:}|{{%s}|{%s}}' % (label, inputlabels, outputlabels)
```

Comment:
Now the line is too long (86 char > 80). Let's break it down

```
label = '{%s}|{input:|output:}|{{%s}|{%s}}' % (
    label, inputlabels, outputlabels)
```

Reply:
Sorry about that, fixed it already.